sql/tables/CMECreditQuestionBinding.sql

```diff
@@ -1,7 +1,16 @@
+drop table CMECreditQuestionBinding;
+
 create table CMECreditQuestionBinding (
+	id serial,
 	credit integer not null references CMECredit(id) on delete cascade,
 	question integer not null references InquisitionQuestion(id) on delete cascade,
 	displayorder integer not null default 0,
 
-	primary key(credit, question)
+	primary key(id)
```

Comment:
This confused me as using an explicit `id` on a binding table is not our norm, but this mirrors `InquisitionInquistionQuestionBinding` so I think it's fine. I'll assume past us were smart about it.